src/tree/tree.scss

```diff
@@ -108,10 +108,13 @@ $iui-expander-button-width: ($iui-icons-default) + ($iui-expander-inline-padding
   }
 
   &.iui-active {
-    @include themed {
-      background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
-      outline: $iui-active-outline;
-      outline-offset: -1px;
+    &,
+    &:focus:not(:focus-visible) {
```

Comment:
This will break all active styling in Safari
https://css-tricks.com/one-invalid-pseudo-selector-equals-an-entire-ignored-selector/

And I don't think it's even correct as now there is no way to tell if the active node is focused.

Reply:
Not sure how to fix it in Sarai?

But you can still see a thicker outline when focusing with keyboard navigation on an active node
![image](https://user-images.githubusercontent.com/88331924/151208880-acec64ea-106e-4c1b-a8d2-6dc7d046b0bd.png)

Reply:
Oh ok, I missed this part:
![image](https://user-images.githubusercontent.com/9084735/151219068-8f3cea2e-17b0-45fd-bc0d-19996377b713.png)

You can try using the `focus-visible` mixin I created in a recent PR:
![image](https://user-images.githubusercontent.com/9084735/151219942-ffaee2f6-6144-469b-a01e-cac0d7662ae6.png)
![image](https://user-images.githubusercontent.com/9084735/151219970-2bae3e9b-3666-4c6e-a70f-36efc16a74a4.png)

Reply:
Actually you could also use the `iui-focus` mixin directly
![image](https://user-images.githubusercontent.com/9084735/151220960-e23aa78b-e7ae-4381-a76e-7da452e15f9e.png)
![image](https://user-images.githubusercontent.com/9084735/151221007-2f2b0249-5454-452c-b64a-ed052250f27f.png)

Reply:
Ok changed to `iui-focus`